tests/Tests/ClientConcepts/LowLevel/PostData.doc.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable IDE1006 // Naming Styles
+#pragma warning disable IDE0044 // Add readonly modifier
```

Comment:
Please move these `#pragma`s below the license header

Reply:
Resolved